web/css/link.css

```
@@ -135,35 +135,34 @@
   text-align: center;
 }
 
-#social-share .fa-facebook {
+#social-share .facebook {
   background-color: #3b5998;
 }
 
-#social-share .fa-twitter {
+#social-share .twitter {
   background-color: #00aced;
 }
 
-#social-share .fa-reddit {
+#social-share .reddit {
   position: relative;
   color: #ff5700;
   background-color: #fff;
   font-size: 48px;
   height: 46px;
   width: 46px;
-  margin-left: 1px;
 }
 
-#social-share .fa-reddit::before {
+#social-share .reddit::before {
   border-radius: 50%;
   position: absolute;
   left: -1px;
-  top: 0;
+  top: -1px;
```

This seems to help with the Reddit icon in Chrome. It's strange that's the only social icon out of the four affected.
```suggestion
  top: -2px;
```